components/pages/home/Navbar.js

```diff
@@ -0,0 +1,191 @@
+import Image from "next/image";
+import { Icon } from "@iconify/react";
+import Link from "next/link";
+import { useState } from "react";
+
+const Navbar = () => {
+  const [isDarkMode, setisDarkMode] = useState(false);
+  const [navbarResponsive, setNavbarResponsive] = useState(false);
+
+  return (
+    <nav className="bg-white drop-shadow mx-auto">
+      <div className="hidden xl:block max-w-[1920px] mb-40 py-5 px-10">
+        <div className="flex justify-between">
+          <div className="flex items-center space-x-16">
+            <div>
+              <Link href="/">
+                <a className="flex items-center">
+                  <Image
+                    src={"/img/bellshade-logo.png"}
+                    alt="Bellshade Logo"
+                    width={"178px"}
+                    height={"48px"}
+                  />
+                </a>
+              </Link>
+            </div>
+            <div className="space-x-7">
+              <Link href="/">
+                <a className="hover:text-main">
+                  <Icon
+                    icon="ic:round-school"
+                    width={24}
+                    height={24}
+                    className="inline mr-3"
+                  />
+                  Learning Jurney
+                </a>
+              </Link>
+              <Link href="/">
+                <a className="hover:text-main">
+                  <Icon
+                    icon="mdi:sword-cross"
+                    width={24}
+                    height={24}
+                    className="inline mr-3"
+                  />
+                  Challenges
+                </a>
+              </Link>
+              <Link href="/">
+                <a className="hover:text-main">
+                  <Icon
+                    icon="ic:baseline-leaderboard"
+                    width={24}
+                    height={24}
+                    className="inline mr-3"
+                  />
+                  Leaderboard
+                </a>
+              </Link>
+              <Link href="/">
+                <a className="hover:text-main">
+                  <Icon
+                    icon="ri:team-fill"
+                    width={24}
+                    height={24}
+                    className="inline mr-3"
+                  />
+                  Team
+                </a>
+              </Link>
+            </div>
+          </div>
+          <div className="flex items-center space-x-8">
+            <div className="flex items-center">
+              <button
+                onClick={() => setisDarkMode(!isDarkMode)}
```

Comment:
Biar sama namanya kayak setternya

```suggestion
                onClick={() => setIsDarkMode(!isDarkMode)}
```